src/routes/availability/[slug]/+page.server.ts

```diff
@@ -47,6 +53,48 @@ const getAvailability = async (
   return availability.sort((a, b) => (a.meeting_dates.date > b.meeting_dates.date ? 1 : -1));
 };
 
+/**
+ * Get all availabilities of members for a meeting
+ *
+ * @param meetingId
+ * @returns a record of the member name to their availabilities, each sorted by date
+ */
+async function getMeetingMemeberAvailabilities(meetingId: string) {
```

Comment:
```suggestion
async function getMeetingMemberAvailabilities(meetingId: string) {
```
typo: Mem**e**ber instead of Member; also change line 34 where this function is called